Director/Director.js

```diff
@@ -0,0 +1,3230 @@
+    </ul>
+  </p>
+  <p>Only works if the current page name contains: <i>scene, stage, theater, theatre</i></p>
+
+  <h4>Wipe Scene</h4>
+  <p><span style="${css.headerButton}; float:none; position:relative;">Wipe the Scene</span> removes all placed images and stops all audio.</p>
+  <p>Only functions on valid stage pages.</p>
+
+  <h4>Edit Mode</h4>
+  <p><span style="${css.headerButton}; float:none; position:relative;">${Pictos(')')}</span> toggles editing. When enabled:</p>
+  <ul>
+    <li>Rename, delete, and move controls appear for acts, scenes, and images</li>
+    <li>Items are grouped by type and display reassign/delete controls</li>
+  </ul>
+    <h4>JB+</h4>
+  <p>If you have the Jukebox Plus script installed, this button will display and will put a link in chat for opening that program's controls.</p>
+    <h4>Help</h4>
+  <p>Displays this Help documentation. While in help mode, this changes to read "Exit Help".</p>
+    <h4>Make Help Handout</h4>
+  <p>This button appears only while in Help mode. Pressing it will create a handout containing the help documentaiton. Useful if you want to see the documentation and the interface at the same time.</p>
+
+
+  <h3>Helpful Macros</h3>
+  <p>The interface is primary, but the following macros can be used in chat or action buttons:</p>
+  <pre style="background:#eee; padding:6px; font-size:12px;">
+!director --set-scene
+!director --wipe-scene
+!director --new-act|Act I
+!director --new-scene|Act I|Opening Scene
+!director --capture-image
+  </pre>
+</div>
+`;
+
+const getJukeboxPlusHandoutLink = () => {
+ const css = getCSS();
+  if (typeof API_Meta !== 'undefined' &&
+      API_Meta.JukeboxPlus &&
+      typeof API_Meta.JukeboxPlus.offset === 'number') {
+    
+    const handout = findObjs({ type: 'handout', name: 'Jukebox Plus' })[0];
+    if (handout) {
+      const url = `http://journal.roll20.net/handout/${handout.id}`;
+      return `<a href="${url}" style="${css.headerButton}"><span style="${css.forceTextColor}">JB+</span></a>`;
+    }
+  }
+  return '';
+};
+
+
+const renderFilterBarInline = (css) => {
+  const st = getState();
+  const activeFilter = st.items?.filter || 'all';
+  const mode = st.settings?.mode || 'light';
+  const borderColor = mode === 'dark' ? '#eee' : '#444';
+
+  // Build dynamic option strings
+  const characters = findObjs({ _type: 'character' }).sort((a, b) => a.get('name').localeCompare(b.get('name')));
+  const handouts = findObjs({ _type: 'handout' }).sort((a, b) => a.get('name').localeCompare(b.get('name')));
+  const macros = findObjs({ _type: 'macro' }).sort((a, b) => a.get('name').localeCompare(b.get('name')));
+  const tables = findObjs({ _type: 'rollabletable' }).sort((a, b) => a.get('name').localeCompare(b.get('name')));
+  const tracks = findObjs({ _type: 'jukeboxtrack' }).sort((a, b) => a.get('title').localeCompare(b.get('title')));
+
+  const buildOpts = (objs, labelFn = o => o.get('name')) =>
+    objs.map(o => `${labelFn(o)},${o.id}`).join('|');
```

Comment:
If the object's name contains a double quote, ie: `Thomas "Tommy" Boy` it will cause the button's href to terminate early. If we escape or replace the `"` with `&quot;` here in buildOpts it should avoid this issue.

```suggestion
    objs.map(o => `${labelFn(o).replace(/"/g, "&quot;")},${o.id}`).join('|');
```